### ISSUE TITLE
Some of Iridium text translations are broken

### ISSUE THREAD

**Issue by yu-str** [iridium-browser/iridium-browser#20]
Some of Iridium text translations are broken. 
Chromium uses Google's localisation tools which in turn use hashes of message_ids as keys in translation dictionaries. That is why it is not enough just to change base text and translated text, we also need to match keys(hashes) since after change of base text its hash doesn't match the translation key.

**Comment by longsleep** [comment]
We should disable all text translations in Iridium and have Iridium only in English for now.

**Comment by jengelh** [comment]
I do not think we should - just like in gettext systems, only the untranslatable string would switch to the original language, not the entire program.
And for Iridium, the fix is rather easy: update the hash IDs. Only need to find the algorithm/the ID of the new string.
